app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardPresenter.kt

```diff
@@ -50,7 +58,12 @@ class ConceptCardPresenter @Inject constructor(
       .newBuilder<SubtitledHtml>()
       .registerViewDataBinderWithSameModelType(
         inflateDataBinding = ConceptCardExampleViewBinding::inflate,
-        setViewModel = ConceptCardExampleViewBinding::setSubtitledHtml)
+        setViewModel = ::bindExampleViewModel)
       .build()
   }
+
+  private fun bindExampleViewModel(binding: ConceptCardExampleViewBinding, html: SubtitledHtml) {
+    val htmlParser = htmlParserFactory.create("skill", skillId)
+    binding.html = htmlParser.parseOppiaHtml(html.html, binding.example)
```

Comment:
We shouldn't call parseOppiaHtml in multiple places--can we consolidate this & the version in view model such that we only bind in one location? That would be slightly cleaner since it de-duplicates the code and makes it easier to reason about.

Reply:
I consolidated heading, explanation, and worked examples into one recycler view, so that all the binding is in one place.